specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2019-08-02/HybridCompute.json

```diff
@@ -0,0 +1,1255 @@
+                "description":"The machine extension instance view."
+             }
+          },
+          "description":"Describes the properties of a Machine Extension."
+       },
+       "MachineExtensionUpdateProperties": {
+         "properties": {
+           "forceUpdateTag": {
+             "type": "string",
+             "description": "How the extension handler should be forced to update even if the extension configuration has not changed."
+           },
+           "publisher": {
+             "type": "string",
+             "description": "The name of the extension handler publisher."
+           },
+           "type": {
+             "type": "string",
+             "description": "Specifies the type of the extension; an example is \"CustomScriptExtension\"."
+           },
+           "typeHandlerVersion": {
+             "type": "string",
+             "description": "Specifies the version of the script handler."
+           },
+           "autoUpgradeMinorVersion": {
+             "type": "boolean",
+             "description": "Indicates whether the extension should use a newer minor version if one is available at deployment time. Once deployed, however, the extension will not upgrade minor versions unless redeployed, even with this property set to true."
+           },
+           "settings": {
+             "type": "object",
+             "description": "Json formatted public settings for the extension."
+           },
+           "protectedSettings": {
+             "type": "object",
+             "description": "The extension can contain either protectedSettings or protectedSettingsFromKeyVault or no protected settings at all."
+           }
+         },
+         "description": "Describes the properties of a Machine Extension."
+       },
+       "MachineExtensionInstanceView":{
+          "properties":{
+             "name":{
+                "type":"string",
+                "description":"The machine extension name."
+             },
+             "type":{
+                "type":"string",
+                "description":"Specifies the type of the extension; an example is \"CustomScriptExtension\"."
+             },
+             "typeHandlerVersion":{
+                "type":"string",
+                "description":"Specifies the version of the script handler."
+             },
+             "status":{
+                "properties":{
+                   "code":{
+                      "type":"string",
+                      "description":"The status code."
+                   },
+                   "level":{
+                      "type":"string",
+                      "description":"The level code.",
+                      "enum":[
+                         "Info",
+                         "Warning",
+                         "Error"
+                      ],
+                      "x-ms-enum":{
+                         "name":"StatusLevelTypes",
+                         "modelAsString":false
```

Comment:
Suggest to set true in case of extending level enum values in the future.

Reply:
since no validation would occur if we set it to true, i'll leave it as false for now.